.travis.yml

```diff
@@ -5,4 +5,5 @@ services:
   - postgresql
 
 before_script:
-  - psql -c 'create database travis_ci_test;' -U postgres
+  - psql -c "CREATE DATABASE langmatetest;" -U postgres
```

Comment:
i think you edit this file on another PR so why you are pushing edit also on this PR?!